js/src/lib/components/InferenceWidget/widgets/TextGenerationWidget/TextGenerationWidget.svelte

```diff
@@ -56,19 +56,20 @@
 		const [textParam] = getSearchParams(["text"]);
 		if (textParam) {
 			setTextAreaValue(textParam);
-			getOutput();
+			getOutput({ useCache });
 		} else {
 			const [demoText] = getDemoInputs(model, ["text"]);
 			setTextAreaValue(demoText ?? "");
 			if (text && callApiOnMount) {
-				getOutput({ isOnLoadCall: true });
+				getOutput({ isOnLoadCall: true, useCache: true });
```

Comment:
**Before:**
TextGen widget onLoad inference call had headers:
```
'X-Load-Model': '0'
'X-Use-Cache': 'false'
```

**With this PR:**
TextGen widget onLoad inference will have headers (without `'X-Use-Cache': 'false'`):
```
'X-Load-Model': '0'
```